### DISCUSSION TITLE
Exterior/Math/P%26S/1/

### DISCUSSION THREAD

**Discussion by giscus** [SAVIA7582/Exterior#11] [General]
# Exterior/Math/P%26S/1/

https://savia7582.github.io/Exterior/Math/P%26S/1/

**Comment by Enxulansis** [comment]
全概率与贝叶斯当中的例1（求患病概率那题）的第二小问，独立测验两次，两次检测都是阳性的概率是P(A₁|B)*P(A₂|B)*P(B)+P(A₁|Bbar)*P(A₂|Bbar)*P(Bbar)=0.95²×0.001+0.002²×0.999而不是P(A)²吧🤔

**Reply by Enxulansis** [reply]
全概率与贝叶斯当中的例8，题干当中最后一句“遇到等级分高的玩家，小王胜，平，负的概率分别0.5，
0.3，0.2. ”应该为“等级分低”？（应该是小错误，不影响理解🤔
